pkg/controller/releasemanager/plan/syncRevision.go

```diff
@@ -167,6 +167,16 @@ func (p *SyncRevision) Apply(ctx context.Context, cli client.Client, cluster *pi
 		}
 	}
 
+	if p.Worker != nil && p.Replicas == 0 {
```

Comment:
Not your fault, but using replicas == 0 as a flag for deployment feels yucky now